src/data/links.yaml

```
@@ -1750,3 +1750,39 @@ links:
       Wish you could avoid airport queues, tedious transfers and long drives?
       Want to travel sustainably and enjoyably by train instead - to fantastic accommodation in the best ski resorts in the Alps?
       Snowcarbon is an independent guide founded by ski journalist Daniel Elkan to help you do exactly that.
+
+  - title: Runamics Running & Sports Gear
+    url: https://www.runamics.com
+    countries: []
+    categories: [clothing]
+    tags: [clothing]
+    description: Crowd-funded for-purpose eco-friendly alternatives to traditional sports gear.  We combine merino wool with wood-based cellulosic fibres like Lyocell. We make experiments with highly innovative fibres like biodegradable elastic fibres or “polyester-like” substitutes.  We're a climate neutral company- We avoid emissions where we can and where they can't be avoided, we compensate them. We will repair your runamics garment and send it back to you for free or discount a replacement. We use plastic-free packaging and ship globally with DHL GoGreen.
+      
+  - title: MakeSense Jobs
+    url: https://jobs.makesense.org
+    countries: []
+    categories: [volunteering, jobs]
+    tags: [career, job, volunteering, internship]
+    description: Makesense inspires and empowers citizens, entrepreneurs and organizations to build an inclusive and sustainable society together.  MakeSense's Job and assignment announcements platform for associations and social entrepreneurs who recruit profiles passionate about the impact. Job types include internship, freelance, civic service, partner, program, volunteer mission, mentoring.
+
+  - title: Loop A Life 100% circular fashion - truly circular and wasteless wear
```

```suggestion
  - title: Loop A Life - 100% Circular Fashion
```

This might also be too much, but its better.

---

Is it worth making a case for optional subtitles? There's plenty of links on the site whose product/company names aren't super straightforward.  What if we added a ~50char human-readable description of what the thing is?